recipes/openms-meta/meta.yaml

```diff
@@ -0,0 +1,229 @@
+        - {{ pin_compatible('eigen', max_pin='x.x') }}
+        - {{ pin_compatible('glpk', max_pin='x.x') }}
+        - {{ pin_compatible('hdf5', max_pin='x.x') }}
+        - {{ pin_compatible('bzip2', max_pin='x.x') }}
+        - {{ pin_compatible('qt', max_pin='x.x') }}
+        - {{ pin_compatible('libsvm', max_pin='x.x') }}
+        - {{ pin_compatible('coinmp', max_pin='x.x') }}
+        - {{ pin_compatible('sqlite', max_pin='x.x') }}
+    test:
+      commands:
+        - OPENMS_DATA_PATH=${PREFIX}/share/OpenMS/ ${PREFIX}/bin/FileMerger --help  
+
+  - name: pyopenms
+    version: {{ version }}
+    script: install_pyopenms.sh
+    # problem with pyopenms is: We could in theory skip CMake configuration
+    # and save a lot of build dependencies BUT then env.py
+    # does not contain the right paths anymore, since the build folder is copied for each
+    # subpackage.
+    requirements:
+      build:
+        - {{ compiler('cxx') }}
+        - cmake
+      host:
+        - {{ pin_subpackage('libopenms', exact=True) }}
+        - llvm-openmp # [osx]
+        - zlib
+        - libtool
+        - xerces-c
+        - boost
+        - eigen
+        - glpk
+        - hdf5
+        - bzip2
+        - qt >=5.5
+        - libsvm
+        - seqan-library ==1.4.2
+        - coinmp
+        - sqlite
+        - python {{ python }}
+        - cython
+        - autowrap
+        - numpy
+        - pip
+        - nose
+      run:
+        - {{ pin_subpackage('libopenms', exact=True) }}
+        - {{ pin_compatible('llvm-openmp', max_pin='x.x') }} # [osx]
+        - {{ pin_compatible('zlib', max_pin='x.x') }}
+        - {{ pin_compatible('xerces-c', max_pin='x.x') }}
+        - {{ pin_compatible('boost', max_pin='x.x') }}
+        - {{ pin_compatible('eigen', max_pin='x.x') }}
+        - {{ pin_compatible('glpk', max_pin='x.x') }}
+        - {{ pin_compatible('hdf5', max_pin='x.x') }}
+        - {{ pin_compatible('bzip2', max_pin='x.x') }}
+        - {{ pin_compatible('qt', max_pin='x.x') }}
+        - {{ pin_compatible('libsvm', max_pin='x.x') }}
+        - {{ pin_compatible('coinmp', max_pin='x.x') }}
+        - {{ pin_compatible('sqlite', max_pin='x.x') }}
+        - python {{ python }}
+        - numpy
+    test:
+      commands:
+        - python -c "import pyopenms; print(pyopenms.__version__)"
+        # We cant use the PYTHON variable since it does not exist on the biocontainer?!
+
+  - name: openms-thirdparty
+    version: {{ version }}
+    requirements:
+      run:
+        - {{ pin_subpackage('openms', exact=True) }}
+        - comet-ms ==2016013 
+        - crux-toolkit ==3.2 
+        - fido ==1.0 
+        - luciphor2 ==2018_06_28 
+        - msgf_plus ==2017.07.21 
+        - omssa ==2.1.9 
+        - pepnovo ==20101117 
+        - percolator ==3.4 
+        - sirius-csifingerid ==4.0.1 
+        - thermorawfileparser ==1.2.1 
+        - xtandem ==15.12.15.2 
+        - gnuplot
+
```

Comment:
Really like this. 

For Galaxy wrappers I currently also inlcude 

- bumbershoot 3_0_11579 (for myrimatch)
- tpp 5.0.0 (for SpectraSTAdapter)

For you reference here is a list of requirements that is not yet in conda: https://github.com/bioconda/bioconda-recipes/issues/18953 .. essentially only MaraCluster is left (and a few which will not make it to conda due to licensing)

Reply:
Ah, that's what they are for, thanks, I will add them! Yes, we also figured it would be great if there is an all-in package. And with the new structure, it should be up to the user to choose which parts they need.
Release will hopefully be tomorrow if everything works out.